Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM node:18.19.0-alpine3.18@sha256:c0a5f02df6e631b75ee3037bd4389ac1f91e591c5c1e30a0007a7d0babcd4cd3 as builder
+
+# Get git
+RUN apk add --no-cache git
+
+# Installaton
+WORKDIR /source
+COPY ./scripts/ /source/scripts/
+COPY ./src/ /source/src/
```

Comment:
The source folder is required just to `yarn install`?  Usually this at least can be put off until after dependency installs, and it is arguably the most important folder to delay copying as much as possible to avoid cache busting.